README.md

```diff
@@ -1384,6 +1386,56 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual meter service
+
+This service enables a device to report accumulated consumption in case it does not have its own metering capabilities (eg. a relay with output binary switch service or thermostat). Thanks to the service accumulated consumption is calculated based on manually provided consumption in a unit of time (eg. watts, m3/h). A device shall calculate cumulated consumption at every state's change, mode change and at least every interval value - 30 minutes by default. When this service is added on a device, appropriate metering service is created (eg. meter_elec, meter_water) and a device starts to send measurements.
+
+#### Service names
+
+`meter_virtual`
+
+#### Interfaces
+
+Type | Interface                | Value type |   Unit  | Description
+-----|--------------------------|------------|---------|------------
+in   | cmd.set_interval         | int        | minutes | Interval  for accumulated consumption recalculation. Overwrites a default value.
```

Comment:
@BudzikM Every single interface we implement has three segments and two dots:
* cmd/evt
* service name or logical component of the service
* action prefixed if applicable with set and get and suffixed if applicable with report

You're not following this convention. Interfaces should be like this:
* cmd.(virtual_)meter.set_interval
* cmd.(virtual_)meter.add(_consumption)
* cmd.(virtual_)meter.remove(_consumption)

Reply:
I cross-checked and it's not everywhere the same. Why should we duplicate cmd.SERVICE_NAME.sth since service's name is already in the field before ? I believe every protocol should be succinct wihtout repetitions.

Reply:
eg:
```
"type": "cmd.add",
"serv": "meter_virtual",

Reply:
Two reasons:

1) Convention. All existing documented interfaces consist of three segments. The reason we have conventions in software development is that it allows a reader/consumer/developer to quickly establish patterns and base his future expectations on these patterns, treating them as learning heuristics. If you break those conventions and start using free-form interface names, this will have a detrimental effect on the entire development and user experience and increase the risk of bugs. Unless there is a damn good reason to stray off the path, even if convention is discussable, it's better to have any convention than none at all.

![image](https://user-images.githubusercontent.com/31538120/180392293-4902d367-fb20-4704-b1c1-08feb888af2c.png)

2) Service component. The middle segment is not a copy of the name of the service, is the name of a component of the service or a logical part of it or a sub-domain of it. Just take the first service from this specification. You see there a config domain and others.

Please add to the interfaces the middle segment like `meter`.

Reply:
ok.